src/standard_library/array_methods.md

```diff
@@ -43,7 +43,7 @@ fn main() {
 Sorts the array with a custom comparison function
 
 ```rust
-fn sort_via<T, N>(mut a: [T; N], ordering: fn(T, T) -> bool) -> Self
+fn sort_via<T, N>(mut a: [T; N], ordering: fn(T, T) -> bool)
```

Comment:
Sorry for the confusion, by removing self, I meant we should replace it with the actual type being returned. In this case `[T; N]`. The same goes for `sort`.